docs/semgrep-app/scm.md

```diff
@@ -1,65 +1,72 @@
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Integrating Semgrep into source code management (SCM) tools
+
+Semgrep App's Team and Enterprise tiers support repositories hosted on both **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. This feature supports all GLSM tiers (Free, Premium, or Ultimate). Create multiple integrations in different self-hosted or custom servers.
+
+## Prerequisites
+
+You need the following permissions to integrate Semgrep into GHE or GLSM servers:
+
+* Permission to create a personal access token (PAT) for the repository to scan Semgrep on. This PAT is used to enable Semgrep App to create pull request (PR) or merge request (MR) comments for findings that can be [autofixed](../notifications/#automatically-fix-your-findings-through-pull-or-merge-requests).
+* Permission to add CI/CD secrets into your [GitHub](https://docs.github.com/en/actions/security-guides/encrypted-secrets) or [GitLab](https://docs.gitlab.com/ee/ci/secrets/) environments.
+
+## Integrating Semgrep into GitHub Enterprise or GitLab Self Managed 
+
+Integrate Semgrep into these custom source code management (SCM) tools by following the steps below:
+
+1. Sign in to [Semgrep App](https://semgrep.dev/login).
+2. Click **[Settings](https://semgrep.dev/orgs/-/settings)** > **Create new parent organization**.
+
+<div class = "bordered">
+
+![Screenshot of settings for parent organization](../img/app-parent-org.png "Screenshot of settings for parent organization")
+
+</div>
+
+3. Confirm the name of your organization. If the provided name is incorrect, please contact r2c at support@r2c.dev 
+4. Sign out, then sign in again to enable SCM management.
+5. Click **Settings** > **SCM management** > **Configure new SCM**.
+
+<div class = "bordered">
+
+![Screenshot of SCM configuration tab](../img/app-scm.png)
+
+</div>
+
+6. Select your SCM provider:
+
+<ol type="A">
+    <li>For <strong>GitHub Enterprise</strong> follow these steps:</li>
+    <ol>
+        <li>Create a personal access token (PAT) by following the steps outlined in this <a href="https://docs.github.com/en/enterprise-server@3.1/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token">guide to creating a PAT</a>.</li>
+        <li>Enter the personal access token generated into the Access Token field.</li>
+        <li>Enter your GHE base URL into the SCM Base URL.</li>
+   </ol>
+    <li>For <strong>GitLab Self Managed</strong>, follow these steps:</li>
+    <ol>
+        <li>Create a personal access token (PAT) by following the steps outlined in this <a href='https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html'>guide to creating a PAT</a>.</li>
+        <li>Enter the personal access token generated into the Access Token field.</li>
+        <li>Enter your GLSM base URL into the <strong>SCM Base URL</strong> field.</li>
+    </ol>
+</ol>
```

Comment:
@s-santillan feel free to make this a normal list again. Need to ask for Docusaurus support...